test/test_manifest.json

```diff
@@ -4784,5 +4784,14 @@
        "rounds": 2,
        "lastPage": 1,
        "type": "fbf"
-    }
+    },
+    {  "id": "pr12564",
+      "file": "pdfs/pr12564.pdf",
+      "md5": "8033857922eb03d050f6a04c2f5e9851",
+      "rounds": 1,
+      "firstPage": 1,
+      "lastPage": 1,
```

Comment:
When you're adding the test-case to the repository (as opposed to linking it), and it *only* contains one page, you don't really need to add those lines :-)

Reply:
sure. Fixed.